tests/test_process.py

```diff
@@ -101,3 +101,15 @@ def test_fetch_nuget_specific(self, tmpdir, nuget, name, version, expected_diges
         assert digest == expected_digest
         assert osp.exists(osp.join(str(tmpdir), '{}.{}.nupkg'.format(name.lower(),
                                                                      version)))
+
+    @pytest.mark.parametrize('name, version, expected_digest', [
+        ('github.com/gorilla/mux', '3f19343c7d9ce75569b952758bd236af94956061',
```

Comment:
@msrb forked mux [1] - I guess not to rely on upstream.

[1] https://github.com/msrb/mux